javascript/index.js

```diff
@@ -1,30 +1,57 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+btnLeft.onclick = function () {
+  if (btnLeft.className === "btn start") {
+    chronometer.startClick(printTime);
+    // printTime as a callback
+    // two functions printMinutes and printSeconds
+  } else if (btnLeft.className === "btn stop") {
+    chronometer.stopClick()
+  }
+};
+const btnRight = document.getElementById("btnRight");
+btnRight.onclick = function () {
+  if (btnRight.className === "btn reset") {
+    chronometer.resetClick()
+  } else if (btnRight.className === "btn split") {
+    chronometer.splitClick()
```

Comment:
I added a function `printSplit` on line 53 which already makes use of your `chronometer.splitClick()` so you can replace it with `printSplit()` and it should work